.github/workflows/docker.build.yml

```diff
@@ -56,4 +56,19 @@ jobs:
           cache-to: type=local,mode=max,dest=/tmp/.buildx-cache
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
-          labels: ${{ steps.docker_meta.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.docker_meta.outputs.labels }}
+      #Create PR on deploy repo from a merged PR
+      -
+        name: create pull request for deploy on staging
```

Comment:
Looks like just running this (without specifying a path) is fine! That surprised me but it looks like this clears the current directory before checking it out (see: https://github.com/rosalieper/ui/actions/runs/3765335384/jobs/6400703190#step:9:28)

Reply:
yep